Packages/com.unity.inputsystem/CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ however, it has to be formatted properly to pass verification tests.
 ### Fixed
 
 - Fixed unclosed profiler marker in `InvokeCallbacksSafe_AnyCallbackReturnsTrue` which would lead to eventually broken profiler traces in some cases like using `PlayerInput` (case ISXB-393).
+- Fixed an issue where connecting a gamepad in the editor with certain settings will cause memory and performance to degrade ([case UUM-19480](https://jira.unity3d.com/browse/UUM-19480)).
```

Comment:
We use public links which will be `https://issuetracker.unity3d.com/product/unity/issues/guid/UUM-19480` in this case

Reply:
Thank you!
(fixed just below in commit a1bf6e1)